foundation-extensions/experience-platform-extension/edge-network-api-reference.md

```diff
@@ -70,9 +69,18 @@ public static void sendEvent(final ExperienceEvent experienceEvent, final EdgeCa
 * _experienceEvent_ - the XDM Experience Event to be sent to Adobe Experience Edge Network
```

Comment:
trying to figure out how to link to the Experience Event in the Java tab.

Reply:
The ExperienceEvent in the Public Classes section? I would suggest creating a Public Classes section with tab groups for each class, similar to the updates to the Identity for Edge Network API reference. Then you can reference the Experience Event section directly, and not an anchor within the tab. If the Java tab is currently selected, it will still be selected when the link is clicked to go to the ExperienceEvent section.
https://aep-sdks.gitbook.io/docs/foundation-extensions/identity-for-edge-network/api-reference#public-classes

Reply:
Same answer for iOS. With ExperienceEvent has a header outside a tab group, you can link directly to the `#experienceevent` anchor which GitBook automatically creates from headers. If the iOS tab is currently selected for `sendEvent`, it will still be selected when using the link to ExperienceEvent.